source/draw/gpu/opengl/OpenGLES3/Context.ooc

```diff
@@ -139,12 +139,12 @@ Context: class {
 		return result
 	}
 	create: static func ~shared (window: NativeWindow, sharedContext: This = null) -> This {
-		Debug print("Creating OpenGL context")
+		version (debugGLv) { Debug print("Creating OpenGL context") }
 		result := This new()
 		result _generate(window, sharedContext) ? result : null
 	}
 	create: static func ~pbufferShared (sharedContext: This = null) -> This {
-		Debug print("Creating OpenGL context")
+		version (debugGLv) { Debug print("Creating OpenGL context") }
```

Comment:
`version (flag)` with space breaks syntax highlighting, so that's probably not right.